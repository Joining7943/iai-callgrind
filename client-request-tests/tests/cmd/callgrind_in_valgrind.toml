bin.name = "valgrind-wrapper"
args = [
  "--tool=callgrind",
  "--valgrind-args=--verbose",
  "--bin=callgrind-reqs-test",
]
status = { code = 1 }
fs.sandbox = true
stdout = ""
