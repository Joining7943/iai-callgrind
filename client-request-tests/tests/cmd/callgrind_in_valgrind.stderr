Zeroing costs...
...done
Start dumping at BB (Client Request)...
Dump to
Dumping done.
Start dumping at BB (Client Request: Please dump here)...
Dump to
Dumping done.
Zeroing costs...
...done
Start dumping at BB (Client Request)...
Dump to
Dumping done.
Start dumping at BB (Client Request: Please dump here)...
Dump to
Dumping done.
Client Request: instrumentation switched OFF
Zeroing costs...
...done
Start dumping at BB (Client Request)...
Dump to
Dumping done.
Start dumping at BB (Client Request: Please dump here)...
Dump to
Dumping done.
Zeroing costs...
...done
Start dumping at BB (Client Request)...
Dump to
Dumping done.
Start dumping at BB (Client Request: Please dump here)...
Dump to
Dumping done.
Client Request: instrumentation switched ON
Zeroing costs...
...done
Start dumping at BB (Client Request)...
Dump to
Dumping done.
Start dumping at BB (Client Request: Please dump here)...
Dump to
Dumping done.
Zeroing costs...
...done
Start dumping at BB (Client Request)...
Dump to
Dumping done.
Start dumping at BB (Client Request: Please dump here)...
Dump to
Dumping done.
Client Request: instrumentation switched OFF
Zeroing costs...
...done
Start dumping at BB (Client Request)...
Dump to
Dumping done.
Start dumping at BB (Client Request: Please dump here)...
Dump to
Dumping done.
Zeroing costs...
...done
Start dumping at BB (Client Request)...
Dump to
Dumping done.
Start dumping at BB (Client Request: Please dump here)...
Dump to
Dumping done.
Client Request: instrumentation switched ON

Start dumping at BB (Prg.Term.)...
Dump to
Dumping done.
Events    : Ir
Collected :

I   refs:
